src/pc/pc_main.c

```diff
@@ -177,8 +177,14 @@ void produce_interpolation_frames_and_delay(void) {
         gfx_end_frame();
 
         // delay
-        if (!configUncappedFramerate) {
-            f64 targetDelta = 1.0 / (f64) configFrameLimit;
+        f64 tarFramerate = 0;
```

Comment:
`tar`?

Reply:
`tar` meant target, albeit it was pretty unclear so I renamed the variable to targetFramerate.